cmd/snap-device-helper/main.c

```diff
@@ -20,14 +20,17 @@
 #include "snap-device-helper.h"
 
 int main(int argc, char *argv[]) {
-    if (argc < 5) {
+    if (argc != 2) {
```

Comment:
shouldn't we ignore the extra args instead for backward compatibility?

Reply:
`tagname` was `argv[2]` and becomes `argv[1]`. What I can do is if `argc > 2`, then I read `argv[2]` for `tagname`. Then we have backward compatibility.

Reply:
Or actually `argc >= 5`.

Reply:
Pushed a fix for it.